examples/8a_benchmark_write_parallel.cpp

```diff
@@ -627,42 +642,41 @@ void AbstractPattern::run()
         std::string filename = s.str();
 
         {
-            std::string tag = "Writing: " + filename;
+            std::string tag = "Writing filebased: " + filename;
             Timer kk(tag.c_str(), m_Input.m_MPIRank);
 
             for (int step = 1; step <= m_Input.m_Steps; step++)
             {
                 setLayOut(step);
                 Series series =
                     Series(filename, Access::CREATE, MPI_COMM_WORLD);
+                series.setIterationEncoding(m_Input.m_Encoding);
                 series.setMeshesPath("fields");
                 store(series, step);
             }
         }
     }
 
-#ifdef NEVER // runs into error for ADIOS. so temporarily disabled
```

Comment:
Was this a bug? Is it resolved now?

Reply:
It was an very old comment, and was resolved.